### PR TITLE
Enhancment/feed screen

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/components/header/view/headerView.js [hunk 1]
```diff
@@ -1,131 +1,107 @@
...
-    );
-  }
-}
+      {isReverse ? (
+        <View style={styles.reverseBackButtonWrapper}>
+          <IconButton
+            style={styles.backButton}
+            iconStyle={styles.backIcon}
+            name="md-arrow-back"
+            onPress={() => handleOnPressBackButton()}
```
Comment (mistikk) [inline src/components/header/view/headerView.js]:
```suggestion
            onPress={handleOnPressBackButton}
```

File: src/components/posts/view/postsView.js [hunk 2]
```diff
@@ -345,76 +331,68 @@ class PostsView extends Component {
...
+      <FlatList
+        data={posts}
+        showsVerticalScrollIndicator={false}
+        renderItem={({ item }) =>
+          get(item, 'author', null) && (
+            <PostCard isRefresh={refreshing} content={item} isHideImage={isHideImage} />
+          )
+        }
+        keyExtractor={(content, i) => `${get(content, 'permlink', '')}${i.toString()}`}
+        onEndReached={() => _loadPosts()}
```
Comment (mistikk) [inline src/components/posts/view/postsView.js]:
```suggestion
        onEndReached={_loadPosts}
```

File: src/components/posts/view/postsView.js [hunk 3]
```diff
@@ -345,76 +331,68 @@ class PostsView extends Component {
...
+        removeClippedSubviews
+        refreshing={refreshing}
+        onRefresh={_handleOnRefreshPosts}
+        onEndThreshold={0}
+        initialNumToRender={10}
+        ListFooterComponent={_renderFooter}
+        onScrollEndDrag={_handleOnScroll}
+        ListEmptyComponent={_renderEmptyContent}
+        refreshControl={
+          <ThemeContainer>
```
Comment (mistikk) [inline src/components/posts/view/postsView.js]: dont use theme container inside of refresh control, android has a bug about it. you can use outside of flatlist

File: src/screens/launch/screen/launchStyles.js [hunk 4]
```diff
@@ -6,4 +6,10 @@ export default EStyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
   },
+  darkContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#1e2835',
```
Comment (mistikk) [inline src/screens/launch/screen/launchStyles.js]: use theme color

**Comment by ue** [comment]
@mistikk updated

**Pull request merged**